modules/swarmNode.js

```diff
@@ -86,14 +86,23 @@ class SwarmNode extends EventEmitter {
         return this.startUsingGateway();
     }
 
+    stop() {
+        if (!this._swarm) {
+            return Q.reject(new Error('Swarm not initialized. Have you called swarmNode.init()?'));
```

Comment:
nit: the error verbiage looks like logging for your debugging purposes. I'd leave out the implementation detail (`swarmNode.init()`) for production.